kit/src/main/java/com/oracle/javafx/scenebuilder/kit/fxom/FXOMSaver.java

```diff
@@ -52,14 +51,14 @@
 class FXOMSaver {
     
     
-    public String save(FXOMDocument fxomDocument) {
+    public String save(FXOMDocument fxomDocument, boolean wildcardImports) {
```

Comment:
Can we extract `wildcardImports` to a class field and pass it to the constructor? That way, you don't need to pass the boolean to each method.